scripts/publish_homebrew_version.sh

```diff
@@ -1,43 +1,48 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-set -e
+set -Eeuf -o pipefail
+set -x
 
-if [[ -z "$VERSION" ]]; then
-  echo "Missing target VERSION environment variable, please specify it"
-  exit 1
-fi
+main() {
+  if [[ -z "${VERSION}" ]]; then
+    echo "Missing target VERSION environment variable, please specify it"
+    exit 1
+  fi
 
-# Removing the v suffix, if present
-VERSION=${VERSION#"v"}
+  # Removing the v suffix, if present
+  local VERSION=${VERSION#"v"}
 
-rm -Rf target/homebrew
-mkdir -p target/homebrew/artifacts
+  rm -rf -- target/homebrew
+  mkdir -p target/homebrew/artifacts
 
-echo "Targeting version $VERSION"
-echo "Downloading macOS artifacts"
+  echo "Targeting version ${VERSION}"
+  echo "Downloading macOS artifacts"
 
-gh release download v$VERSION --pattern "Espanso-Mac*" --dir target/homebrew/artifacts
+  gh release download v"${VERSION}" --pattern "Espanso-Mac*" --dir target/homebrew/artifacts
 
-echo "Reading artifacts hashes"
-INTEL_SHA=$(cat target/homebrew/artifacts/Espanso-Mac-Intel.zip.sha256.txt | awk -F ' ' '{print $1}')
-M1_SHA=$(cat target/homebrew/artifacts/Espanso-Mac-M1.zip.sha256.txt | awk -F ' ' '{print $1}')
+  echo "Reading artifacts hashes"
+  SHA256=$(awk -F ' ' '{print $1}' target/homebrew/artifacts/Espanso-Mac-Universal.zip.sha256.txt)
 
-echo "Cloning tap repository"
+  echo "Cloning tap repository"
 
-pushd target/homebrew
-git clone git@github.com:espanso/homebrew-espanso.git
+  pushd target/homebrew
+  git clone git@github.com:espanso/homebrew-espanso.git
```

Comment:
This is another thing, I believe we are not using the tap anymore.
I have very little knowledge of how to upload a package to homebrew. If my memory serves, we are submitting packages directly to the main hombrew repo

Reply:
I'm no longer using homebrew at all (nix + nix-darwin convert), but I'm happy to get this working as I know this will be a major path for installation on macos!

Reply:
Mmmm, now that I peek again at it, the prod-ci _also_ pushes to the hombrew cask, and not the main repository.
Let's put this to a side and resolve it in another PR, it's not a problem for today